crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1051,6 +1051,9 @@ spec:
                             items:
                               type: object
                               properties:
+                                auto_enable:
+                                  description: "Enabled option by default"
```

Comment:
OK, after looking at what this does, I agree "auto_enable" doesn't really make sense because you aren't selecting a boolean checkbox here - these are just selecting the drop-down expressions for the find/hide text field.

`is_default` wouldn't be good. It's not really a default. All of these could be set to "true" (where as a "is_default" value implies its one value of many and only one is selected to be "the default").

"auto_apply" seems to make sense.

This is essentially auto-selecting the drop-down option. Would "auto_select" make more sense?

Reply:
I like "auto_select" because it implies that you should set this to "true" on only one option.  The description should probably mention this as well, and also note that the default is "false".  @josunect , if you are OK with "auto_select" then please make the change.

Reply:
> also note that the default is "false"

I like to have defaults explicitly set in the defaults.yml only because it makes things more explicit when the ConfigMap is generated. But if you prefer not to see these set in the ConfigMap, we can remove these from the defaults.yml file that I asked for earlier.

Reply:
How does this sound?

```suggestion
                                auto_select:
                                  description: "If true this option will be selected and take effect automatically. Note that only one option in the list can have this value be set to true."
```